docs/intern-projects/prescriptions-bootstrap.md

```diff
@@ -0,0 +1,52 @@
+# Bootstrap database of Thoth prescriptions
+
+Assigned intern:
+Assigned mentor:
+
+## Project Goal
+
+The goal of this project is to extend the current database of prescriptions
+with known or not yet known issues in the Python ecosystem.
+
+## Deliverables
+
+Set of YAML files committed to
+[thoth-station/prescriptions](https://github.com/thoth-station/prescriptions)
+repository that help the recommender system to advise better software stacks to
+users.
+
+## Prerequisites for Team Members
+
+Please check you have all the following:
+
+* Access to Red Hat network
```

Comment:
is this required due to access to stage?

Reply:
It might be not required, will remove it. Thanks. 👍🏻